.gitignore

```diff
@@ -9,6 +9,8 @@ node_modules
 /util/python/python_lib
 /pip_test
 /_python_build
+**/tools/git/gen
```

Comment:
I'm just not sure it should be in the general .gitignore. You can just add this to your local `.git/info/exclude`